modules/accredited_representative_portal/app/policies/accredited_representative_portal/saved_claim_claimant_representative_policy.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class SavedClaimClaimantRepresentativePolicy < ApplicationPolicy
+    def index?
+      authorize
+    end
+
+    private
+
+    def authorize
+      @user.user_account.active_power_of_attorney_holders.size.positive?
+    end
+
+    class Scope < ApplicationPolicy::Scope
+      def resolve
+        @scope.for_power_of_attorney_holders(
+          @user.user_account.active_power_of_attorney_holders
+        )
+      end
+    end
```

Comment:
Interesting. I was under the impression that, for now, the set of claims submissions that are visible on this page are the ones that had been created by _this user_ in the past, rather than those that had been created by any of the VSOs they currently belong to. I suppose we could further restrict that to their current POA holder memberships.

As such, I think the query would be based on the following fields:
```
- power_of_attorney_holder_type
- power_of_attorney_holder_poa_code
- accredited_individual_registration_number
```

And if for some reason we did want to let them see old submissions from past VSOs, we could only compare against `accredited_individual_registration_number`.

I think that some corresponding class method query helper does seem appropriate.

Reply:
As for that bundle of external IDs, it looks like an example in the growing case for some more encapsulation around these things. It's good that we're gathering the evidence for it now.